src/GraphQL/Validation/Rules/FieldsOnCorrectType.cs

```diff
@@ -64,7 +63,7 @@ private static IEnumerable<string> GetSuggestedTypeNames(IGraphType type, string
             if (type is IAbstractGraphType absType)
             {
                 var suggestedObjectTypes = new List<string>();
-                var interfaceUsageCount = new LightweightCache<string, int>(key => 0);
+                var interfaceUsageCount = new Dictionary<string, int>();
```

Comment:
LightweightCache just wraps a dictionary... one less allocation every time this runs...